plugin/restore/mysql/mysql_test.go

```diff
@@ -256,20 +256,6 @@ func TestGetLatestBackupBeforeOrEqualTsImpl(t *testing.T) {
 			},
 			err: false,
 		},
-		// backup with empty binlog info does not count
```

Comment:
Please add a new test case for the change.

Reply:
I restore this test.